src/transport_interface/README.md

```diff
@@ -44,6 +46,13 @@ The transport interface tests verify the implementation by running various test
 |TransportRecv_RemoteDisconnect	|Test transport interface receive function return value when disconnected by remote server	|Negative value should be returned	|
 |TransportRecv_NoDataToReceive	|Test transport interface receive function return value when no data to receive	|0 should be returned	|
 |TransportRecv_ReturnZeroRetry	|Test transport interface receive function return zero due to no data to receive. Send data to echo server then retry the receive function. Transport receive function should be able to receive data from echo server and return positive value.	|Postive value should be returned after retry transport receive	|
+|TransportWritev_NetworkContextNullPtr    |Test transport interface writev with NULL network context pointer handling       |Negative value should be returned      |
+|TransportWritev_BufferNullPtr    |Test transport interface writev with NULL transport vector array pointer handling       |Negative value should be returned      |
+|TransportWritev_ZeroByteToSend    |Test transport interface writev with zero vector array length to send handling  |Negative value should be returned      |
+|Transport_WritevOneByteRecvCompare    |Test writev receive behavior in the following order<br>Send : 1 byte<br>Send : ( TRANSPORT_TEST_BUFFER_WRITABLE_LENGTH - 1 ) bytes<br>Receive : TRANSPORT_TEST_BUFFER_WRITABLE_LENGTH bytes | Send/receive/compare should has no error |
+|Transport_WritevRecvCompare    |Test transport interface with writev, receive and compare on bulk of data.<br>The data size ranges from 1 byte to TRANSPORT_TEST_BUFFER_WRITABLE_LENGTH bytes |Send/receive/compare should has no error within timeout |

```

Comment:
I'm fairly certain "Send/receive/compare should **has** no **error**" should be "Send/receive/compare should **have** no **errors**"
I'm not going to let it hold up a review, especially since it's a common copy/paste in this table. But I do think it's _technically_ incorrect.